src/components/search/results/EventList.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import EventItem from './EventItem';
+
+const EventList = () => {
+  return (
+    <div className="event-list-Container">
```

Comment:
You might want to see about refactoring this to use module CSS like the above example